packages/rpc/src/methods/account.ts

```diff
@@ -1,7 +1,7 @@
 import { Network } from '@mintbase-js/sdk';
 import { RPC_OPTIONS, requestFromNearRpc } from '../util';
 
-export const accountExists = async (accountId: string, network?: Network, rpc?: RPC_OPTIONS): Promise<boolean> => {
+export const accountExists = async (accountId: string, network?: Network, rpc?: RPC_OPTIONS, rpcUrl?: string): Promise<boolean> => {
```

Comment:
This opens up an edge case where someone could supply `network = 'mainnet'`, but a testnet RPC URL